src/OrleansSQLUtils/Storage/DbStoredQueries.cs

```diff
@@ -192,14 +192,18 @@ internal static ReminderEntry GetReminderEntry(IDataRecord record)
             /// </summary>
             private static string TryGetSiloName(IDataRecord record)
             {
-                for (var i = 0; i < record.FieldCount; ++i)
```

Comment:
@xclayl In addition, there's always room for improvements. If you notice something and aren't in a hurry, just a make a mental note and drop it to a chat or open an issue and someone and eventually things improve. :)

I think I have have some time tomorrow to take a deeper look at this, but I need to upgrade my VM with PostregreSQL so realistically I drop comments, if there's any &ndash; looks good at a glance &ndash; on Wednesday or Thursday. If there's hurry to get into the 1.3.0 time window I can go to sleep a bit later. :) @jdom? @sergeybykov? Still time?